compiler/il/ILOpCodeProperties.hpp

```diff
@@ -2037,6 +2037,134 @@
    /* .ifCompareOpCode      = */ TR::BadILOp,
    },
 
+   {
+   /* .opcode               = */ TR::fmadd,
+   /* .name                 = */ "fmadd",
+   /* .properties1          = */ 0,
+   /* .properties2          = */ ILProp2::ValueNumberShare | ILProp2::SupportedForPRE,
+   /* .properties3          = */ ILProp3::LikeUse,
+   /* .properties4          = */ 0,
+   /* .dataType             = */ TR::Float,
+   /* .typeProperties       = */ ILTypeProp::Size_4 | ILTypeProp::Floating_Point,
+   /* .childProperties      = */ THREE_SAME_CHILD(ILChildProp::UnspecifiedChildType),
```

Comment:
I believe the types of children for these opcodes should be `THREE_SAME_CHILD(TR::Float)` and `THREE_SAME_CHILD(TR::Double)`, as appropriate.

Reply:
done